tests/components/climate/test_demo.py

```diff
@@ -237,6 +237,13 @@ def test_set_aux_heat_bad_attr(self):
         self.hass.block_till_done()
         self.assertEqual('off', state.attributes.get('aux_heat'))
 
+    def test_set_aux_heat_on_ecobee(self):
+        """Test setting the axillary heater on/true."""
+        climate.set_aux_heat(self.hass, True, ENTITY_ECOBEE)
```

Comment:
Ecobee does not support `set_aux_heat`, you would need to turn aux heat on by setting the operation to `auxHeatOnly`.